src/appsignal/config.py

```diff
@@ -96,100 +44,218 @@ class Options(TypedDict, total=False):
     working_directory_path: Optional[str]
 
 
-CA_FILE_PATH = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "resources", "cacert.pem"
-)
-
-DEFAULT_CONFIG = Options(
-    ca_file_path=CA_FILE_PATH,
-    enable_host_metrics=True,
-    enable_nginx_metrics=False,
-    enable_statsd=False,
-    endpoint="https://push.appsignal.com",
-    files_world_accessible=True,
-    send_environment_metadata=True,
-    send_params=True,
-    send_session_data=True,
-)
+class Config:
```

Comment:
Consider [adding type hints](https://stackoverflow.com/a/47161395) for the instance properties:
```suggestion
class Config:
  initial_source: Optional[Options]
  system_source: Options
  environment_source: Options
  options: Options
```